src/EnergyPlus/OutputReportPredefined.cc

```diff
@@ -1972,25 +1972,25 @@ namespace OutputReportPredefined {
 
 		//results from regular zone and system sizing calcs, "At Ideal Loads Peak"
 		pdch2CoilDDnameSensIdealPeak = newPreDefColumn( pdst2CoilSummaryCoilSelection, "Design Day Name at Sensible Ideal Loads Peak" );
-		pdch2CoilDateTimeSensIdealPeak = newPreDefColumn( pdst2CoilSummaryCoilSelection, "Date/Time at Sensible Ideal Loads Peak" );
+		pdch2CoilDateTimeSensIdealPeak = newPreDefColumn( pdst2CoilSummaryCoilSelection, "Date/Time at Sensible Ideal Loads Peak {TIMESTAMP}" );
```

Comment:
@JasonGlazer So, adding {TIMESTAMP} here is causing TableDiff to [crash](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-e7ccfff88223a238bd5d66cc022f021f5c48a4e7-x86_64-MacOS-10.9-clang.html). Can you see anything wrong here?

Reply:
Nothing jumps out as to why this would cause a problem.